static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -609,13 +637,24 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                         variants: rows.map(row => ({
                             id: row.name,
                             products: [],
+                            ...(isLiveBlog
+                                ? { test: setupEpicInLiveblog }
+                                : {}),
                             options: {
                                 buttonTemplate: isThankYou
                                     ? undefined
                                     : defaultButtonTemplate,
                                 locations: splitAndTrim(row.locations, ','),
                                 tagIds: splitAndTrim(row.tagIds, ','),
                                 sections: splitAndTrim(row.sections, ','),
+                                excludedTagIds: optionalSplitAndTrim(
```

Comment:
New columns in the sheet should in general be optional, otherwise it's very easy to break things when copying from older versions

Reply:
shall we make all the array ones optional? just for extra safety